Pages/wishlist.html

```diff
@@ -0,0 +1,262 @@
+                    <div class="star-rating">
+                        <i class="rating fas fa-star starred"></i>
+                        <i class="rating fas fa-star starred"></i>
+                        <i class="rating fas fa-star starred"></i>
+                        <i class="rating fas fa-star starred"></i>
+                        <i class="rating fas fa-star"></i>
+                    </div>
+                    <div class="card-horizontal-footer">
+                        <button class="btn btn-primary-solid">
+                            <i class="fas fa-heart"></i> Move to Cart
+                        </button>
+                        <button class="btn btn-secondary-outline">
+                            <i class="fas fa-trash"></i> Remove from Wishlist
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </main>
+
+    <!-- Footer -->
+
+    <footer class="flex-align-center">
+
+        <div class="contact">
+            <p><i class="icons fas fa-phone-alt"></i> +91 0123456789</p>
+            <p><i class="icons fab fa-whatsapp"></i> +91 9876543210</p>
+            <p><i class="icons far fa-envelope"></i> support@urbantails.com</p>
+        </div>
+
+        <div class="footer-section">
+
+            <div class="about"><span>About Us</span>
+                <p>Our Story</p>
+                <p>Blogs</p>
+                <p>Careers</p>
+            </div>
+
+            <div class="about"><span>Support</span>
+                <p>FAQ</p>
+                <p>Contact Us</p>
+                <p>Return & Refund Policy</p>
+                <p>Shipping Information</p>
+                <p>Payment Options</p>
+            </div>
+
+            <div class="about"><span>Keep in touch!</span>
+                <form class="form">
+                    <input class="input" type="email" name="email-id" placeholder="Enter your email address" />
+                    <button class="btn btn-error-solid btn-squared">
+                        <i class="fas fa-door-open"></i> Log In
+                    </button>
+                </form>
+            </div>
+        </div>
+
+        <ul class="social-links list-non-bullet">
+            <li class="list-item" ;>
+                <a href="https://github.com/AshutoshM29"> <i class="icons-social fab fa-github"></i></a></li>
+            <li class="list-item">
+                <a href="https://twitter.com/Astomi_29?s=08"> <i class="icons-social fab fa-twitter"></i></a></li>
+            <li class="list-item">
+                <a href="https://www.instagram.com/the.astomi/"> <i class="icons-social fab fa-instagram"></i></a></li>
+            <li class="list-item">
+                <a href="https://www.linkedin.com/mwlite/in/ashutosh-mishra-b50423206"><i
+                        class="icons-social fab fa-linkedin"></i></a></li>
+
+        </ul>
+        <div class="with-love">Made with ❤️ by : Ashutosh</div>
```

Comment:
suggestion : you can use icon here for more flexibility if needed

Reply:
ok, I'll keep that in mind